selftests/checkall

```diff
@@ -72,6 +89,7 @@ else
     [ ! $SELF_CHECK_CONTINUOUS ] && CMD+=" --failfast on"
     run_rc selftests "$CMD"
 fi
+[ "$AVOCADO_RESULTSDIR_CHECK" ] && run_rc job-results results_dir_content "$RESULTS_DIR_CONTENT"
```

Comment:
This is probably unrelated to this work, and I have to confirm what's going on here, but this is what I got on one execution:

```
All checks PASSED
selftests/check_tmp_dirs
('There are temporary avocado dirs lying around after test: %s', ['avocado___main__zorKtA'])
```

Which is weird, as the `check_tmp_dirs` should be considered in "All checks", that is, it should not read "PASSED".